src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/CleanUpRejectedFilesTask.java

```diff
@@ -53,58 +56,67 @@ public void run() {
         log.info("Started {} job", TASK_NAME);
 
         blobManager
-            .listRejectedContainers()
-            .forEach(container -> {
-                log.info("Scanning for old rejected files in container {}", container.getName());
-                container
-                    .listBlobs(null, true, EnumSet.of(SNAPSHOTS), null, null)
-                    .forEach(listItem -> {
-                        tryDeleteFile(container, listItem);
-                    });
-            });
+            .getRejectedContainers()
+            .forEach(this::cleanUpContainer);
 
         log.info("Finished {} job", TASK_NAME);
     }
 
-    private void tryDeleteFile(CloudBlobContainer container, ListBlobItem listItem) {
-        final String zipFileName = FilenameUtils.getName(listItem.getUri().toString());
+    private void cleanUpContainer(BlobContainerClient containerClient) {
```

Comment:
cleanUpContainer ? are you deleting files here then worth naming it accordingly ?

Reply:
deleting files in that container. Same method  as Blob router.

Reply:
`deleteFilesInRejectedContainer`